ordering/components/ProductPriceSpec.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { pipe, map } from 'ramda'
+import { connect as connectFela } from 'react-fela'
+import { reduxForm, Field } from 'redux-form'
+import { TextField } from 'redux-form-material-ui'
+import h from 'react-hyperscript'
+
+import { FormattedMessage } from '../../lib/Intl'
+
+import styles from '../styles/ProductPriceSpec'
+
+function ProductPriceSpec (props) {
+  const { styles, priceSpec } = props
+
+  return (
+    h('div', {
+      className: styles.container
+    }, [
+      h(FormattedMessage, {
+        id: 'ordering.priceSpec',
+        className: styles.priceSpecText,
+        values: {
```

Comment:
this is rad, but shouldn't we pass in `priceSpec.currency` and use that as well in the internationalization message (instead of `$`, which only applies to currencies that use name dollar)? if we want to use a symbol, keen to have a hard-coded object that maps currencies to symbols, like https://github.com/root-systems/cobuy/blob/master/quantity/data/units.js, then use that.